src/components/Tooltip/Tooltip.js

```diff
@@ -115,7 +117,14 @@ export const Tooltip = ({
                 {children}
               </div>
             ) : (
-              <div className={styles.icon} ref={ref} {...referenceProps}>
+              <div
+                role="button"
+                className={styles.icon}
+                ref={ref}
+                aria-labelledby="info-tooltip"
+                tabIndex={0}
```

Comment:
The tabIndex paired with the onFocus and onBlur are what allow users who rely solely on the keyboard to navigate the screen to be able to interact with this element